### DISCUSSION TITLE
a timer linked to a habit tracker

### DISCUSSION THREAD

**Discussion by AtifJamel** [iSoron/uhabits#1476] [Feature Requests]
I'd like to see this feature added to the app, which is to add a haibts tracker for specific tasks with a timer attached to it, so that if I usually read for 15 minutes, the app will know that if I click on the haibts tracker of reading, after 15 minutes it will notify me that the time is finished and register it automatically. This will be a really useful feature.

**Comment by c-mena** [comment]
Related to #2296
